docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -114,6 +114,11 @@ Following the OpenVINO™ naming convention, the Automatic Device Selection mode
 | |                              | | Indicates the priority for a model.                                |
 | |                              | | IMPORTANT: This property is not fully supported yet.               |
 +--------------------------------+----------------------------------------------------------------------+
+| | `ov::execution_devices`      | | Lists the runtime target devices on which the inferences are being |
+| |                              | | executed.                                                          |
+| |                              | | Examples of returning results could be `CPU_HELP`, `CPU`, `GPU`,   |
```

Comment:
Please add explanation for CPU_HELP situation. 
 
CPU_HELP as output name will be updated. Please follow up.

Reply:
Added it, please review.